web/tests/functional/products/test_products.py

```diff
@@ -168,6 +168,8 @@ def test_get_product_data(self):
                          os.path.basename(self.test_workspace),
                          "The displayed name must == the default value, as "
                          "we didn't specify a custom displayed name.")
+        confidentiality = pr_conf.confidentiality
+        self.assertEqual(confidentiality, "Confidential classified")
```

Comment:
Can you please modify the confidentality in these test cases, get it again from the server and see whether it is changed or not.